com.pokemmo.PokeMMO.yaml

```diff
@@ -0,0 +1,80 @@
+app-id: com.pokemmo.PokeMMO
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk
+command: pokemmo.sh
+tags:
+  - proprietary
+finish-args:
+- --socket=x11
+- --share=ipc
+- --socket=wayland
+- --socket=pulseaudio
+- --device=all
+- --share=network
+- --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+
+modules:
+  - shared-modules/libdecor/libdecor-0.2.0.json
+  - name: glfw
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
+      - -DBUILD_SHARED_LIBS:BOOL=ON
+      - -DGLFW_USE_WAYLAND=ON
+    sources:
+      - type: git
+        url: https://github.com/glfw/glfw.git
+        commit: 3eaf1255b29fdf5c2895856c7be7d7185ef2b241
+    cleanup:
+      - /include
+      - /lib/cmake
+      - /lib/pkgconfig
+      - /app/share/doc/GLFW/
+
+  - name : openjdk
+    buildsystem : simple
+    build-commands :
+      - /usr/lib/sdk/openjdk/install.sh
+
+  - name: pokemmo
+    buildsystem: simple
+    build-commands:
+      - install -D pokemmo.sh /app/bin/pokemmo.sh
+      - install -Dm 755 apply_extra /app/bin/apply_extra
+      # Icon
+      - install -Dm644 16x16.png /app/share/icons/hicolor/16x16/apps/com.pokemmo.PokeMMO.png
+      - install -Dm644 32x32.png /app/share/icons/hicolor/32x32/apps/com.pokemmo.PokeMMO.png
+      - install -Dm644 128x128.png /app/share/icons/hicolor/128x128/apps/com.pokemmo.PokeMMO.png
+      # Desktop entry
+      - install -Dp -m 644 com.pokemmo.PokeMMO.desktop /app/share/applications/com.pokemmo.PokeMMO.desktop
+      # Placing appdata
+      - install -Dm644 -t /app/share/metainfo com.pokemmo.PokeMMO.metainfo.xml
+    sources:
+      - type: extra-data
+        filename: PokeMMO-Client.zip
+        url: https://dl.pokemmo.com/download/PokeMMO-Client.zip
+        sha256: d8a713105b6263ec26257e22f84c213db8ded32d479ab30a685a838ec8371a47
+        size: 77373135
+        x-checker-data:
+          type: rotating-url
+          url: https://dl.pokemmo.com/download/PokeMMO-Client.zip
+      - type: script
+        dest-filename: apply_extra
+        commands:
+          - unzip /app/extra/PokeMMO-Client.zip -d .
+          - rm -f /app/extra/PokeMMO-Client.zip
+      - type: file
+        path: pokemmo.sh
+      - type: file
+        path: com.pokemmo.PokeMMO.desktop
+      - type: file
+        path: com.pokemmo.PokeMMO.metainfo.xml
+      - type: file
+        path: icons/16x16.png
+      - type: file
+        path: icons/32x32.png
```

Comment:
since they don't come from upstream, these small icons are unecessary and should be removed from here. on 128x128 or higher is.